docs/source/llms/llm-evaluate/index.rst

```diff
@@ -494,6 +497,104 @@ up OpenAI authentication to run the code below.
             model_type="question-answering",
         )
 
+.. _llm-eval-model-endpoint:
+
+Evaluating with a Model Endpoint URL
```

Comment:
We should use an `endpoints:/` URI, not a URL

Reply:
Yea I found many utilities are implemented there and using the endpoints URI simplifies the logic a lot. Thanks for the clarification!